ddpui/api/transform_api.py

```
@@ -1416,3 +1416,34 @@ def get_canvas_node(request, node_uuid: str):
+    """
+    orguser: OrgUser = request.orguser
+    org = orguser.org
+
+    if not org.dbt:
+        raise HttpError(400, "Organization does not have a dbt workspace configured")
+
+    # Get warehouse
+    try:
+        warehouse_obj = OrgWarehouse.objects.get(org=org)
+    except OrgWarehouse.DoesNotExist:
+        raise HttpError(400, "Organization does not have a warehouse configured")
```

Ruff (0.14.7)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)